data/tribes/initialization/empire/units.lua

```diff
@@ -1638,7 +1642,9 @@ wl.Descriptions():new_tribe {
             -- TRANSLATORS: Lore author helptext for an Empire production site: Armorsmithy
             lore_author = pgettext("empire_building", "An armorsmith teaching an apprentice"),
             -- TRANSLATORS: Purpose helptext for an Empire production site: Armor Smithy
-            purpose = pgettext("empire_building", "Forges armor and helmets that are used for training soldiers’ health in the training camp.")
+            purpose = pgettext("empire_building", "Forges armor and helmets that are used for training soldiers’ health in the training camp."),
+            -- TRANSLATORS: Performance helptext for an Empire production site: Armor Smithy
+            performance = pgettext("empire_building", "If this building is fully supplied and all armors are needed by the economy, production of one helmet takes %1$s and one armor, one chain armor and one gilded armor takes %2$s on average. If only one kind of armor is needed by the economy, production of one helmet takes %3$s, one armor, one chain armor or one gilded armor takes %4$s on average."):bformat(format_minutes_seconds(3, 13), format_minutes_seconds(6, 25), format_minutes_seconds(1, 11), format_minutes_seconds(1, 21))
```

Comment:
"2 helmets and one of each armor"?

Reply:
@tothxa I don't understand this comment

Reply:
Same issue as Amazon stone workshop / similar to smokeries and smelters.

Reply:
```suggestion
            performance = pgettext("empire_building", "If this building is fully supplied and all armors are needed by the economy, production of one helmet takes %1$s and one of each kind of armor takes %2$s on average. If only helmet is needed by the economy, its production takes %3$s. If only one kind of armor is needed by the economy, its production takes %4$s on average."):bformat(format_minutes_seconds(3, 13), format_minutes_seconds(6, 25), format_minutes_seconds(1, 11), format_minutes_seconds(1, 21))
```

Reply:
```suggestion
            performance = pgettext("empire_building", "If this building is fully supplied and all kinds of armor and helmets are needed by the economy, production of one helmet takes %1$s and one of each kind of armor takes %2$s on average. If only one kind of armor or helmets is needed by the economy, production of one helmet takes %3$s, while one kind of armor takes %4$s on average."):bformat(format_minutes_seconds(3, 13), format_minutes_seconds(6, 25), format_minutes_seconds(1, 11), format_minutes_seconds(1, 21))
```

Reply:
> ```suggestion
>            performance = pgettext("empire_building", "If this building is fully supplied and all kinds of armor and helmets are needed by the economy, production of one helmet takes %1$s and one of each kind of armor takes %2$s on average. If only one kind of armor or helmets is needed by the economy, production of one helmet takes %3$s, while one kind of armor takes %4$s on average."):bformat(format_minutes_seconds(3, 13), format_minutes_seconds(6, 25), format_minutes_seconds(1, 11), format_minutes_seconds(1, 21))
>```

But there's only one kind of helmet.

Reply:
yes there is only one kind of helmets. I believe kind of goes with a form of plural and armor is just an exception as it can be used in a plurlaized meaning describing a spezies of items rather then a single item